src/consensus/ethash/mod.rs

```diff
@@ -219,4 +207,19 @@ impl Consensus for Ethash {
             vec![]
         })
     }
+
+    fn validate_header_seal(&self, header: &BlockHeader) -> Result<(), DuoError> {
```

Comment:
```suggestion
    fn validate_header_parallel(&self, header: &BlockHeader) -> Result<(), DuoError> {
```

also add `needs_parallel_validation(&self) -> bool` defaulting to `false`